packages/node_modules/@webex/internal-plugin-conversation/test/integration/spec/create.js

```diff
@@ -15,49 +15,47 @@ describe('plugin-conversation', function () {
   this.timeout(60000);
   describe('#create()', () => {
     let checkov, kirk, mccoy, participants, webex, spock;
+    let sampleTextOne = 'sample-text-one.txt';
 
-    before(() => testUsers.create({count: 4})
-      .then(async (users) => {
-        [spock, mccoy, checkov, kirk] = users;
-        participants = [spock, mccoy, checkov];
-
-        // Pause for 5 seconds for CI
-        await new Promise((done) => setTimeout(done, 5000));
+    before(async () => {
```

Comment:
I see you sneaky refactoring!

Reply:
YES!
There was an `async` function as the promise callback 🤔